index.html

```diff
@@ -1,231 +1,373 @@
+                <button type="submit" class="btn-success">Get in touch</button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact-me">
+        <h1>Contact</h1>
+        <div class="contact-wrap">
+          <div class="grid-col-1">
+            <p>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </p>
+          </div>
+          <div class="grid-col-2">
+            <form action="https://formspree.io/f/xyyoovqy" method="post">
+              <div class="user-name">
+                <div class="text-input full-name">
+                  <label for="fullname">Name</label>
+                  <input
+                    type="text"
+                    name="fullname"
+                    placeholder="John Doe"
+                    id="fullname"
+                    required
+                    maxlength="30"
+                  />
                 </div>
-            </div>
-            
-        </section>
+              </div>
 
-        <footer id="footer-social">
-            <ul>
-                <li><img src="./assets/images/github.svg" alt="" /></li>
-                <li><img src="./assets/images/linkedin.svg" alt="" /></li>
-                <li><img src="./assets/images/angelist.svg" alt="" /></li>
-                <li><img src="./assets/images/twitter.svg" alt="" /></li>
-                <li><img src="./assets/images/medium.svg" alt="" /></li>
-            </ul>
-        </footer>
-    </div>
+              <div class="text-input user-email">
+                <label for="useremail">Email</label>
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="john.doe@mail.com"
+                  id="useremail"
+                  required
+                />
+              </div>
+              <div class="text-input user-message">
+                <label for="message">Message</label>
+                <textarea
+                  id="message"
+                  name="user-message"
+                  placeholder="Enter your message here..."
+                  maxlength="500"
+                  required
+                ></textarea>
+              </div>
+              <div>
+                <button type="submit" class="btn-success">Get in touch</button>
+              </div>
+            </form>
+          </div>
+        </div>
+      </section>
 
+      <footer id="footer-social">
+        <ul>
+          <li><img src="../assets/images/github.svg" alt="github" /></li>
+          <li><img src="../assets/images/linkedin.svg" alt="linkedin" /></li>
+          <li><img src="../assets/images/angelist.svg" alt="angelist" /></li>
+          <li><img src="../assets/images/twitter.svg" alt="twitter" /></li>
+          <li><img src="../assets/images/medium.svg" alt="medium" /></li>
+        </ul>
+      </footer>
```

Comment:
- [ ] Please work on these icons and make them visible
-
![b](https://user-images.githubusercontent.com/63972670/173923751-6ee1c078-de29-4b0c-861c-4350bd850a24.PNG)